cni/network/network.go

```diff
@@ -742,6 +764,13 @@ func (plugin *NetPlugin) createEndpointInternal(opt *createEndpointInternalOpt)
 	}
 
 	if opt.resultV6 != nil {
+		// inject ipv6 routes to Linux pod
+		ipamMode := string(util.IpamMode(opt.nwCfg.IPAM.Mode))
+		if ipamMode == "dualStackOverlay" {
+			epInfo.IPV6Mode = ipamMode
```

Comment:
i guess instead of checking ipammode, we should set executionmode to dualstackswift and set that in `epInfo.ipamMode`